datadog-opentelemetry/src/core/configuration/remote_config.rs

```diff
@@ -857,18 +858,25 @@ impl ProductRegistry {
     }
 }
 
-/// Extract product name and id from remote config path
-/// Path format is: datadog/2/{PRODUCT}/{config_id}/config
+/// Extract product and id from remote config path
+/// Path format is: ^(datadog/\d+|employee)/[^/]+/[^/]+/[^/]+$
```

Comment:
would be interesting to add a test?

Reply:
I'll do a follow up PR later with more tests and even better parsing 😬